pkg/apis/wksprovider/controller/wksctl/machine_actuator.go

```diff
@@ -164,11 +165,25 @@ func (a *MachineActuator) create(ctx context.Context, cluster *clusterv1.Cluster
 // don't miss any updates. The plan is derived from the original seed node plan and stored in a config map
 // for use by the actuator.
 func (a *MachineActuator) initializeMasterPlanIfNecessary(installer *os.OS) error {
-	master, err := a.getMasterNode() // Only one can exist at this point
+
+	// we also use this method to mark the first master as the "originalMaster"
```

Comment:
This is a _much_ nicer way to track original master 👍

Reply:
thank you!